### ISSUE TITLE
Why *0.5 the loss?

### ISSUE THREAD

**Issue by cassie101** [bearpaw/pytorch-pose#90]
Hi @bearpaw thank you for sharing the work, may I know what does the 0.5 stand for in the loss function? 
`loss += 0.5 * self.criterion(heatmap_pred, heatmap_gt)`

**Comment by jake-shomer** [comment]
Same question

**Comment by v1093** [comment]
Here is an explanation: https://datascience.stackexchange.com/questions/52157/why-do-we-have-to-divide-by-2-in-the-ml-squared-error-cost-function
